beacon_node/client/src/builder.rs

```diff
@@ -413,7 +413,7 @@ where
                 let blobs = if block.message().body().has_blobs() {
                     debug!("Downloading finalized blobs");
                     if let Some(response) = remote
-                        .get_blobs::<E>(BlockId::Root(block_root), None, &spec)
+                        .get_blob_sidecars::<E>(BlockId::Root(block_root), None, &spec)
```

Comment:
This endpoint is being deprecated and will likely be removed soon - we can probably address this in a separate PR

Reply:
We're handling this gracefully at the moment so it's not a problem, i think we might eventually just remove this call and download columns from peers.